packages/jest-runner/src/runTest.ts

```diff
@@ -163,6 +163,8 @@ async function runTestInternal(
 
   const start = Date.now();
 
+  config.setupFiles.forEach(path => runtime!.requireModule(path));
```

Comment:
Id o this up here since it happened in the constructor previously. However, I wonder if it makes more sense to do it after we call the environment's `setup` function further down? If not, maybe right before? Seems like they belong together

https://github.com/facebook/jest/blob/9fbe3c5bd07002d569a1b4037d53556244a728cd/packages/jest-runner/src/runTest.ts#L230

Reply:
We can mark it as a todo for later, to group similar features together. However for now, to avoid unintended breakages, I'd stay with what's closer to previous version

Reply:
yeah, makes sense